.ci/Jenkinsfile

```diff
@@ -66,10 +68,42 @@ pipeline {
         }
       }
     }
+    stage('Release') {
+      options { skipDefaultCheckout() }
+      when { tag pattern: '\\d+\\.\\d+\\.\\d+', comparator: 'REGEXP' }
+      steps {
+        dir("${BASE_DIR}"){
+          withGoEnv(pkgs: ['github.com/goreleaser/goreleaser']){
+            withCredentials([string(credentialsId: "${env.GITHUB_TOKEN_CREDENTIALS}", variable: 'GITHUB_TOKEN')]) {
+              // Ensure that tags are present so goreleaser can build the changelog from the last release.
+              gitCmd(cmd: 'fetch', args: '--unshallow --tags')
+              sh(label: 'goreleaser', script: 'goreleaser release')
```

Comment:
@endorama , this should make the trick and avoid using the bash installation in favour of the default go install, that's provided by the `withGoEnv` shared library step

Reply:
As a first step this sounds good! I'll iterate on it after merging. Thanks!